src/components/general/FilterList.js

```diff
@@ -258,7 +258,7 @@ class FilterList extends Component {
 		let regexVal = regexFunc(data);
 		let desiredVal = selectBy(data);
 
-		if (!search) {
+		if (search === undefined || search === null) {
```

Comment:
Isn't this the same as `search == null` or `search == 0`?

Reply:
I don't know exactly what your talking about the main issue here is that search is a boolean that can be true or false. but We want different functionality if it wasn't defined than if it was false.